uint32 user_id
---
uint16[] item_ids
